<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lib_dialog</class>
 <widget class="QDialog" name="Lib_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="MaxY">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextEdit" name="LY">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>2</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextEdit" name="gridY">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextEdit" name="MinY">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>2</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="Graph_widget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100000</width>
       <height>600000</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="MinX">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextEdit" name="LX">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextEdit" name="gridX">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextEdit" name="MaxX">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Graph_widget</class>
   <extends>QWidget</extends>
   <header location="global">graph_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
